statsmodels/tsa/statespace/representation.py

```diff
@@ -105,6 +105,17 @@ class Representation(object):
     initial_variance : float, optional
         Initial variance used when approximate diffuse initialization is
         specified. Default is 1e6.
+    initialization : {'approximate_diffuse','stationary','known'}, optional
+        Initialization method for the initial state.
+    initial_variance : float, optional
```

Comment:
Does initial_variance need to be an array if state is not 1-d? Or is diffusive spherical, s * eye?